docker-entrypoint.sh

```diff
@@ -19,61 +25,90 @@ if [ "${1}" = 'edgedb-server' ] && [ "$(id -u)" = '0' ]; then
 		unset EDGEDB_DATADIR
 	fi
 
-	exec gosu edgedb "$BASH_SOURCE" "$@"
+	exec gosu edgedb env _SKIP_ENTRYPOINT=1 "$BASH_SOURCE" "$@"
 fi
 
-if [ "${1}" = 'edgedb-server' -a "$#" -eq 1 ]; then
+if [[ "$1" = 'edgedb-server' ]] && [[ "$2" = "--bind-address=0.0.0.0" ]] && [[ "$#" -eq 2 ]]; then
 	mkdir -p "${EDGEDB_DATADIR}"
 	chown -R "$(id -u)" "${EDGEDB_DATADIR}" 2>/dev/null || :
 	chmod 700 "${EDGEDB_DATADIR}" 2>/dev/null || :
 
-	if ! [ "$(ls -A ${EDGEDB_DATADIR})" ]; then
+	if [[ -z "$(ls -A ${EDGEDB_DATADIR})" ]]; then
 		shopt -s dotglob
 
+        bootstrap_opts=("--port=5656")
+        edgedb_user="${EDGEDB_USER:-edgedb}"
+        edgedb_database="${EDGEDB_DATABASE:-edgedb}"
+        if [[ -e "/edgedb-bootstrap.edgeql" ]]; then
+            bootstrap_opts+=("--bootstrap-script=/edgedb-bootstrap.edgeql")
+        elif [[ -n "$EDGEDB_BOOTSTRAP_COMMAND" ]]; then
+            bootstrap_opts+=("--bootstrap-command=$EDGEDB_BOOTSTRAP_COMMAND")
```

Comment:
This will break easily if EDGEDB_BOOTSTRAP_COMMAND has quotes in it.  Perhaps echo into file and append to `--bootstrap-script` instead?

Reply:
No it will not. Substitution happens after tokenization. So anything in that variable including quotes will fit into the single array member of `bootstrap_opts` and each member is expanded as a single argument in `"${bootstrap_opts[@]}"` substitution (in the `[@]`-case that quotes don't mean fit everything into a single argument, but rather put every argument into it's own virtual quotes).